components/custom/SearchableSelect/SearchableSelect.svelte

```diff
@@ -78,10 +76,10 @@ const onChange = () => {
     list={randomId}
     placeholder="&nbsp;"
     bind:this={element}
-    bind:value={internalChoice}
+    value={choice}
```

Comment:
If the `choice` input later changes (such as if an API call returns that populates that input data), will this still replace the current selection with the new value in `choice`?

Reply:
Yes, the one way binding directly to choice is still reactive. I added a delay to set choice in the story to confirm this is working.